server/src/test/java/org/elasticsearch/action/admin/cluster/migration/TransportGetFeatureUpgradeStatusActionTests.java

```diff
@@ -36,7 +37,7 @@ public void testGetFeatureStatus() {
             FEATURE
         );
 
-        assertThat(status.getUpgradeStatus(), equalTo(MIGRATION_NEEDED));
+        assertThat(status.getUpgradeStatus(), equalTo(NO_MIGRATION_NEEDED));
```

Comment:
What do you think about adding a test with MIGRATION_NEEDED and using index version 6.x?

Reply:
It's tricky because we don't have a public Version constant for anything before `Version.7_0_0`. I'll see if I can figure something out, though.

Reply:
Yeah, maybe we can construct it with the Version math functions? I just want to make sure we have a test that actually validates we will say migration needed when we need it for real.